src/device.c

```diff
@@ -236,6 +230,9 @@ static int on_driver_poll_receive(int id)
 		return -EINVAL;
 
 	rc = driver->read(data_item_aux);
+	if (rc < 0)
```

Comment:
what does rc means? And this zero, what does the zero means? I think the number zero should be replaced by a variable with a explicit name